server-docs/src/test/java/org/neo4j/doc/server/ExclusiveServerTestBase.java

```diff
@@ -19,33 +19,24 @@
  */
 package org.neo4j.doc.server;
 
-import java.util.concurrent.Callable;
-
 import org.junit.BeforeClass;
 import org.junit.Rule;
 import org.junit.rules.TestName;
 
-import org.neo4j.doc.test.rule.SuppressOutput;
 import org.neo4j.doc.test.rule.TestDirectory;
 
-import static org.neo4j.doc.test.rule.SuppressOutput.suppressAll;
 
 public class ExclusiveServerTestBase
```

Comment:
This group of classes is still duplicated. Can we do something about this?

Reply:
aha, those classes are also copy-pasted( and they are waiting for their cleaning PR